.github/workflows/build.yml

```diff
@@ -466,11 +475,3 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           sync-labels: true
         continue-on-error: true
-      - name: Checkout
-        uses: actions/checkout@v2
-        continue-on-error: true
-      - name: /yarn.lock changes
-        uses: Simek/yarn-lock-changes@main
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-        continue-on-error: true
```

Comment:
did you mean to remove this section?

Reply:
Yes, it's not compatible with Yarn v3 unfortunately.